src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -130,6 +129,19 @@ internal static RuntimeGraph GetRuntimeGraph(IEnumerable targetFrameworks)
                            .Select(ToPackageDownloadDependency));
                 }
 
+                if (cpvmEnabled && targetFrameworkInfo is IVsTargetFrameworkInfo3 targetFrameworkInfo3)
+                {
+                    if (targetFrameworkInfo3.CentralPackageVersions != null)
+                    {
+                        tfi.CentralPackageVersions.AddRange(
+                           targetFrameworkInfo3.CentralPackageVersions
```

Comment:
Can `CentralPackageVersions` be null? If so, I think this could cause an `ArgumentNullException` during the Cast.

Reply:
They should not be null